cinema/tests/test_movie_viewset.py

```diff
@@ -0,0 +1,259 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APIClient
+from rest_framework import status
+
+from cinema.models import Movie, MovieSession, CinemaHall, Genre, Actor
+from cinema.serializers import MovieDetailSerializer, MovieListSerializer
+
+MOVIE_URL = reverse("cinema:movie-list")
+MOVIE_SESSION_URL = reverse("cinema:moviesession-list")
+
+
+def sample_movie(**params):
+    defaults = {
+        "title": "Sample movie",
+        "description": "Sample description",
+        "duration": 90,
+    }
+    defaults.update(params)
+    return Movie.objects.create(**defaults)
+
+
+def sample_genre(**params):
+    defaults = {
+        "name": "Drama",
+    }
+    defaults.update(params)
+    return Genre.objects.create(**defaults)
+
+
+def sample_actor(**params):
+    defaults = {"first_name": "George", "last_name": "Clooney"}
+    defaults.update(params)
+    return Actor.objects.create(**defaults)
+
+
+def sample_movie_session(**params):
+    cinema_hall = CinemaHall.objects.create(
+        name="Blue", rows=20, seats_in_row=20
+    )
+    defaults = {
+        "show_time": "2022-06-02 14:00:00",
+        "movie": None,
+        "cinema_hall": cinema_hall,
+    }
+    defaults.update(params)
+    return MovieSession.objects.create(**defaults)
+
+
+def image_upload_url(movie_id):
+    """Return URL for recipe image upload"""

```

Comment:
The docstring mentions 'recipe image upload', which seems unrelated to the context of movies. Consider updating the docstring to accurately reflect the purpose of the function, which is to return the URL for movie image upload.